examples/multimodal/Vision_Fine_tuning_on_GPT4o_for_Visual_Question_Answering_on_Medical_Images.ipynb

```diff
@@ -0,0 +1,816 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Vision Fine-tuning on GPT-4o for Visual Question Answering on Medical Images"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We're excited to announce the launch of [Vision Fine-Tuning on GPT-4o](https://openai.com/index/introducing-vision-to-the-fine-tuning-api/), a cutting-edge multimodal fine-tuning capability that empowers developers to fine-tune GPT-4o using both **images** and **text**. With this new feature, you can customize models to have stronger image understanding capabilities, unlocking possibilities across various industries and applications.\n",
+    "\n",
+    "From **advanced visual search** to **improved object detection** for autonomous vehicles or smart cities, and even **precision in medical image analysis**, vision fine-tuning enables you to craft solutions tailored to your specific needs. By combining text and image inputs, this product is uniquely positioned for tasks like **visual question answering**, where detailed, context-aware answers are derived from analyzing images.\n",
+    "\n",
+    "In this guide, we’ll walk you through the steps to fine-tune GPT-4o with multimodal inputs. Specifically, we’ll demonstrate how to train a model for answering questions related to **medical radiology images**, but the potential applications span countless domains—from **web design** and **education** to **healthcare** and **research**.\n",
```

Comment:
Something that might be beneficial here is to discuss when to use vision fine-tuning.

We often mention to customers that fine-tuning is great for teaching the model how to structure an output and is not useful for teaching the model new information. How does that stay the same/differs with vision fine tuning?

Reply:
Added a line to address this point!